projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -66,6 +71,50 @@ function run(command, ...args) {
 	});
 }
 
+/**
+ * Prompt the user for input.
+ */
+function prompt(question, options) {
```

Comment:
Obvious candidate for extraction into another file, along with the `log` _et al_ functions.